### PR TITLE
pla-665 Dependency updates

### PR DESCRIPTION
**Pull request by peterabbott**
Part of [pla-665-git-security-warning](https://linear.app/atomic-app/issue/PLA-665/git-security-warning) to update some versions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
-    "@typescript-eslint/parser": "^4.29.1",
+    "@babel/eslint-parser": "7.16.3",
+    "@babel/eslint-plugin": "7.14.5",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.33.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-no-async-foreach": "^0.1.1",
     "eslint-plugin-no-only-tests": "^2.6.0",
     "eslint-plugin-prettier": "^3.4.0",
     "prettier-eslint": "^13.0.0"
```
Comment (peterabbott) [inline package.json:18]: Not sure that prettier-eslint is required. I took it out and nothing seemed to break when run against any of the action-card services (using yarn link)

File: package.json [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
   },
+  "resolutions": {
+    "ansi-regex": "^5.0.1"
+  },
   "peerDependencies": {
-    "babel-eslint": "^10.1.0",
```
Comment (peterabbott) [inline package.json:19]: babel-eslint was deprecated some time ago and replaced by @babel/eslint-parser and -plugin

File: package.json (hunk 1 above)
Comment (gregplaysguitar) [reply package.json:18]: I think this is related to our having to have a prettierrc file in the action-cards repo - in theory we should be able to define prettier rules in our eslint config via this package, but I don't think we've got that working yet.

Fine to remove it for now if it's causing a problem I think?

**Review by gregplaysguitar — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (peterabbott) [reply package.json:18]: I made some progress, working on update to roll this out. 

The only unsure thing related to prettier,  is. how VSCode setup will change for automatic formatting. For the short-term we can leave the prettier config at the parent action-cards folder.

**Pull request merged**